### PR TITLE
#42 1 table component res page

### PR DESCRIPTION
**Pull request by nikitakukharchuk**

### CODE REVIEW

**Comment by nikitakukharchuk** [comment]
Add a table component with incorrect answers on result page.

File: src/components/common/IncorrectAnswers/incorrectAnswers.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import { BasicTable } from './BasicTable';
+
+type IncorrectAnswerPropsType = {
+  IncorrectAnswersList: ResultTableDataType[];
+};
+
+export const IncorrectAnswers: FC<IncorrectAnswerPropsType> = ({
+  IncorrectAnswersList,
+}) => (
+  <div>
```
Comment (gitkoshelew) [inline src/components/common/IncorrectAnswers/incorrectAnswers.tsx]: what this div does?

File: src/components/common/IncorrectAnswers/BasicTable.tsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+            bgcolor: 'primary.main',
+            color: 'primary.contrastText',
+            borderRadius: 2,
+          }}
+        >
+          <TableRow>
+            <TableCell
+              sx={{
+                color: 'primary.contrastText',
+                width: '200px',
```
Comment (gitkoshelew) [inline src/components/common/IncorrectAnswers/BasicTable.tsx]: are you sure that it should be hardcoded? maybe you need ad least media queries?

File: src/components/common/IncorrectAnswers/BasicTable.tsx [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+              display: 'flex',
+              marginBottom: '20px',
+            }}
+          >
+            <div className='border-transparent rounded-3xl bg-background-paperAccent2 mr-24 ml-5'>
+              <TableCell
+                component='th'
+                scope='row'
+                sx={{
+                  width: '100px',
```
Comment (gitkoshelew) [inline src/components/common/IncorrectAnswers/BasicTable.tsx]: also not sure that width should be hardcoded without mediaqueries

Comment (nikitakukharchuk) [reply src/components/common/IncorrectAnswers/BasicTable.tsx]: Also in this place remove hardcore code and add grid

File: src/components/common/IncorrectAnswers/BasicTable.tsx (hunk 2 above)
Comment (nikitakukharchuk) [reply src/components/common/IncorrectAnswers/BasicTable.tsx]: I fix it. Use grid for this components.

File: src/components/common/IncorrectAnswers/incorrectAnswers.tsx (hunk 1 above)
Comment (nikitakukharchuk) [reply src/components/common/IncorrectAnswers/incorrectAnswers.tsx]: Sorry, my mistake. I remove it.

**Review by gitkoshelew — APPROVED** [review]

**Review by Zenater — APPROVED** [review]

**Pull request merged**